use star_line::run;
use star_line::functions::errors::CustomError;

fn main() -> Result<(), CustomError>{
    return run();
}